src/platform/utilities/sso/index.js

```diff
@@ -2,6 +2,7 @@ import moment from 'moment';
 import environment from 'platform/utilities/environment';
 import localStorage from '../storage/localStorage';
 import { hasSession, hasSessionSSO } from '../../user/profile/utilities';
+import { loginAppUrlRE } from 'applications/login/utilities/paths';
```

Comment:
@hinzed1127 I have a feeling its bad form to import an applications module into platform, but `applications/login` seemed like the most logic place to put the RegEx.  Thoughts?

Reply:
it's not common, but this seems to be among the small instances that's reasonable. Having it as an export groups it in a logical place, but I also think just inlining the regex would be ok, too. matter of preference

Reply:
it's not great but there are other imports from applications into platform :/